apps/functions-subscription/api/internal.yaml

```diff
@@ -93,6 +93,39 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProblemJson'
+  /trials/{trialId}/activations:
```

Comment:
**suggestion**: What about renaming this endpoint to reflect more the resource we are going to manipulate? E.g. `/trials/{trialId}/activations-job`